app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, source_name, destination_name = "", "", ""
+    try:
+        cmd, source_name, destination_name = command.split()
+    except Exception:
+        print("Error in cmd!")
+    print(cmd, source_name, destination_name)
+
+    dir_str = ""
+
+    if cmd == "mv":
+        if "/" in destination_name:
+            list_dir = destination_name.split("/")
```

Comment:
Don't use `split` for working with directions, because each OS has personal separator. You should use `os.path.split`